lib/clinic_finder/gestation_helper.rb

```diff
@@ -0,0 +1,28 @@
+class ClinicFinder
+  class GestationHelper
+    attr_reader :gestational_age, :gestational_weeks
+
+    def initialize(gestational_age)
+      @gestational_age = gestational_age
+      @gestational_weeks = (@gestational_age/7.0).ceil
```

Comment:
Again, so we have it explicit, can we make a note about some of the business logic here?